docs/component_guides/evaluation/feedback_aggregation.md

```diff
@@ -11,13 +11,13 @@ aggregation can be used.
         Feedback(provider.context_relevance_with_cot_reasons, name = "Context Relevance")
         .on(Select.RecordCalls.retrieve.args.query)
         .on(Select.RecordCalls.retrieve.rets)
-        .aggregate(np.mean)
+        .aggregate(numpy.mean)
```

Comment:
`np` is pretty commonplace imo. don't feel strongly about it though

Reply:
yeah I only changed to to `numpy` cuz of the description below (I think there are more instances of `numpy.mean` than `np.mean` throughout our doc 🔢 )